### DISCUSSION TITLE
Копирование фото

### DISCUSSION THREAD

**Discussion by Renatik001** [eneshecan/whatsapp-for-linux#286] [Q&A]
В телеграмм нажимаю скопировать фотографию и могу тут же в чат ее вставить (отредактировать и отправить). Нажимаю скопировать в whatsapp, фото не копируется.

![image](https://user-images.githubusercontent.com/131782737/234298333-62a4e031-6f59-44c4-8a64-8e5c526483e6.png)

**Comment by PythonPlumber** [comment]
**В Телеграме я могу нажать кнопку "Копировать" на фотографии и сразу же вставить ее в чат (редактировать и отправить). Однако, когда я нажимаю кнопку "Копировать" в WhatsApp, фотография не копируется и я не могу ее вставить в чат.**

_translated using Yandex._
